### PR TITLE
feat: updating api keys examples

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Update api keys examples 

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/api/authentication.mdx [hunk 1]
```diff
@@ -12,10 +12,10 @@ Basis Theory uses API keys to allow access to the API.
 
 Basis Theory requires the API key to be included in all API requests to the server in a header that looks like the following:
 
-`BT-API-KEY: key_N88mVGsp3sCXkykyN2EFED`
+`BT-API-KEY: <API Key>`
 
 <Alert>
-  You must replace <code>key_N88mVGsp3sCXkykyN2EFED</code> with your API key
+  You must replace <code>&lt;API Key></code> with your API key
```
Comment (mstrisoline) [inline docs/api/authentication.mdx:18]:
```suggestion
  You must replace <code>&lt;API Key&gt;</code> with your API key
```
Should we also use the `&gt;` syntax here to close the `chevrons` referencing a variable here?

**Review by mstrisoline — COMMENTED** [review]
Some general feedback. Should we be using All CAPS w/ Underscores for our placeholder names? IE: `API_KEY` or `PRIVATE_API_KEY` encapsulated in `<>` the `ALL UPPER` is general a standard for variables, especially in shell scripts. The use of `_` also allows you do double click and select the entire placeholder value vs. the separation via space. In general White space isn't in a variable name so it feels very odd when reading this that there is a space between `API` and `Key` etc.

File: docs/api/authentication.mdx (hunk 1 above)
Comment (jleon15) [reply docs/api/authentication.mdx:18]: `>` doesn't need to be escaped in XML

**Review by mstrisoline — DISMISSED** [review]

**Review by mstrisoline — APPROVED** [review]

**Pull request merged**